libfeed/feed-store-provider.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright 2018 Endless Mobile, Inc.
+ *
+ * eos-discovery-feed is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * eos-discovery-feed is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with eos-discovery-feed.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+/**
+ * EosDiscoveryFeedGjsStyleGDBusCallback:
+ * @array: (element-type EosDiscoveryFeedOrderableModel) (transfer none): a #GPtrArray of #EosDiscoveryFeedOrderableModel.
```

Comment:
The name for this is awful - I picked this format for compatibility with `wrapDBusProxy` in gjs